Compilation fails if cdefs has MGOS_ENABLE_WEB_CONFIG: 1

If I set `MGOS_ENABLE_WEB_CONFIG: 1` in `cdefs` then compilation fails with following error:

```
/home/martin/app/deps/http-server/src/mgos_http_server.c: In function 'ro_vars_handler':
/home/martin/app/deps/http-server/src/mgos_http_server.c:146:13: error: 'mgos_sys_ro_vars' undeclared (first use in this function)
   send_cfg(&mgos_sys_ro_vars, mgos_ro_vars_schema(), hm, c);
```

Reference: https://forum.mongoose-os.com/discussion/4262/compilation-fails-if-cdefs-has-mgos-enable-web-config-1

--- a/src/mgos_http_server.c
+++ b/src/mgos_http_server.c
@@ -37,12 +37,9 @@
 #include "mgos_init.h"
 #include "mgos_mongoose.h"
 #include "mgos_net.h"
+#include "mgos_ro_vars.h"
 #include "mgos_sys_config.h"
 #include "mgos_utils.h"
-
-#if MGOS_ENABLE_WEB_CONFIG
-#include "mgos_ro_vars.h"
-#endif
 
 #define MGOS_F_RELOAD_CONFIG MG_F_USER_5
 

--- a/src/mgos_http_server.c
+++ b/src/mgos_http_server.c
@@ -40,6 +40,10 @@
 #include "mgos_sys_config.h"
 #include "mgos_utils.h"
 
+#if MGOS_ENABLE_WEB_CONFIG
+#include "mgos_ro_vars.h"
+#endif
+
 #define MGOS_F_RELOAD_CONFIG MG_F_USER_5
 
 #if MG_ENABLE_FILESYSTEM

